modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserAccountResourceImpl.java

```diff
@@ -292,24 +291,19 @@ public Page<UserAccount> getOrganizationUserAccountsPage(
 			Pagination pagination, Sort[] sorts)
 		throws Exception {
 
-		Organization organization = _organizationResourceDTOConverter.getObject(
-			organizationId);
-
 		return _getUserAccountsPage(
 			_getModelActions(
 				Collections.singletonMap(
 					ActionKeys.MANAGE_USERS,
 					new String[] {"getOrganizationUserAccountsPage"}),
-				organization.getOrganizationId(),
+				GetterUtil.getLong(organizationId),
```

Comment:
Hey @MrJohn1911, please keep using the _organizationResourceDTOConverter since the organizationId parameter may not equal the organization primaryKey in all cases.

Thx